vercel.json

```diff
@@ -2,22 +2,22 @@
   "redirects": [
     {
       "source": "/releases/mainnet/active/contracts.json",
-      "destination": "/releases/mainnet/1.9.1/contracts.json",
+      "destination": "/releases/mainnet/1.9.2/contracts.json",
```

Comment:
In general we first merge a PR which keeps mainnet pointing to the prev version (which is finally changed when the spell is cast).
In this specific case I think we are fine leaving the PR unmerged until the spell actually passes as this is not something that integrators need to use.

Reply:
Should not be changed until mainnet release has been cast. This should be moved out to a separate PR.

Reply:
updating the final bits we need (PIP addresses mainnet, IPFS mainnet, and changing the date delimiter) now that the mainnet spell is `cast()`